tests/unit/logs/test_transformation_full.py

```diff
@@ -98,7 +97,175 @@
+                                "logEvents": [{"id": "36184182057683330175263108764349521965726085205046394887",
+                                               "timestamp": 1622554840009,
+                                               "message": json.dumps({
+                                                   "eventVersion": "1.08",
+                                                   "userIdentity": {
+                                                       "type": "IAMUser",
+                                                       "principalId": "AIDA12345667789ABCEF",
+                                                       "arn": "arn:aws:iam::444000444:user/somemonitoringuser",
+                                                       "accountId": "444000444",
+                                                       "accessKeyId": "AKIA123456789ABCDEFG",
+                                                       "userName": "Dynatrace_monitoring_user"
+                                                   },
+                                                   "eventTime": "2021-05-28T11:39:33Z",
+                                                   "eventSource": "rds.amazonaws.com",
+                                                   "eventName": "DescribeEvents",
+                                                   "awsRegion": "ap-southeast-2",
+                                                   "sourceIPAddress": "155.55.55.055",
+                                                   "userAgent": "aws-sdk-java/1.11.789 Linux/5.4.0-56-generic OpenJDK_64-Bit_Server_VM/11.0.8+10 java/11.0.8 vendor/AdoptOpenJDK",
+                                                   "requestParameters": {
+                                                       "startTime": "May 28, 2021 11:29:00 AM",
+                                                       "endTime": "May 28, 2021 11:34:00 AM"
+                                                   },
+                                                   "responseElements": None,
+                                                   "requestID": "69de2278-ef7a-4b9a-bdc5-a78c9210ea4e",
+                                                   "eventID": "249491df-4970-421d-8575-5bf555357c14",
+                                                   "readOnly": True,
+                                                   "eventType": "AwsApiCall",
+                                                   "managementEvent": True,
+                                                   "eventCategory": "Management",
+                                                   "recipientAccountId": "444000444"
+                                               })
+                                               }
+                                              ]},
+        "expect_first_log_contains": {
+            'aws.service': 'cloudtrail',
+            # "aws.resource.id": "TODO",
```

Comment:
TODOs left here and below

Reply:
Who said it's review ready? ;) that one still waits for commit&push